srcpkgs/Electron-Cash/template

```diff
@@ -14,9 +13,13 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="MIT"
 homepage="https://electroncash.org"
 distfiles="https://github.com/Electron-Cash/Electron-Cash/releases/download/${version}/Electron-Cash-${version}.tar.gz"
-checksum=befcedee8e7d5f4a1f2a012c97d8b0f0b452728145ce9d398a1aec69b7f126c7
+checksum=bfab25545cee3309037aa4849fe418c6f59e5e899aa1de1c73ea05957ce914ea
 
 post_install() {
+	case "$XBPS_TARGET_MACHINE" in
+	arm*|aarch*)
+		rm -f -- "${DESTDIR}/usr/lib/python${py3_ver}/site-packages/electroncash/tor/bin/tor" ;;
+	esac
```

Comment:
You can remove the file unconditionally, then, no need to check for ARM (and in this case the correct check would be to check if not x86). Leave a comment with `# TODO: build these binaries instead of having to remove pre-built ones`

Reply:
will do!